lib/ui/widgets/patcherView/app_selector_card.dart

```diff
@@ -49,11 +53,12 @@ class AppSelectorCard extends StatelessWidget {
                       ),
                     ),
                     const SizedBox(width: 6),
```

Comment:
Newer `screen_util` allows you to make adaptive spaces between widgets, you can use it here.

Reply:
Good to know! This could definitely help with UI consistency, as sometimes, the padding doesn't match.